src/TimeSignature.jsx

```diff
@@ -1,8 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import {range} from './helpers';
```

Comment:
can you remove `range` from helpers too plz?

Reply:
Apparently it's being used somewhere else.
![image](https://user-images.githubusercontent.com/20604769/103688784-a3642180-4f92-11eb-9fd7-70ae139123c8.png)